platform-channels.md

```diff
@@ -405,61 +535,36 @@ Objective-C:
 
 1. Expand Runner > Runner in the Project navigator.
 
-1. If present, delete the files `AppDelegate.h`, `AppDelegate.m`, and
-`Supporting files/main.h`. Select 'Move to trash' when asked.
-
-1. Add a new `AppDelegate.swift` main file:
-  * Right-click the Runner>Runner folder, and select **New File...**. 
-  * Select `Swift File`, and click **Next**
-  * Name the file `AppDelegate.swift`
-  * Select **Create Bridging Header** when asked.
-1. Open `Runner-Bridging-Header.h`, and add a Flutter import:
-    ```
-    #import "../Flutter/Flutter.framework/Headers/Flutter.h"
-    #import "GeneratedPluginRegistrant.h" //for other plugins
-    ```
 1. Open the file `AppDelegate.swift` located under Runner > Runner in the Project
-navigator, and replace `import Foundation` with:
-    ```swift
-    import UIKit
-    import Flutter
-
-    @UIApplicationMain
-    @objc class AppDelegate: FlutterAppDelegate {
-    }
-    ```
-1. Validate that the project builds (shortcut `cmd-b`).
-
-With the project converted to use Swift for the main application class we can
-now begin.
+navigator.
 
-First we override `didFinishLaunchingWithOptions` to contain a
-`FlutterMethodChannel` tied to the channel name `samples.flutter.io/battery`:
+Next, override extend the `application` function create a `FlutterMethodChannel`
+tied to the channel name `samples.flutter.io/battery`:
 
 ```swift
 @UIApplicationMain
 @objc class AppDelegate: FlutterAppDelegate {
   override func application(
     _ application: UIApplication,
     didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
-    GeneratedPluginRegistrant.register(with: self)
-    let controller : FlutterViewController = window?.rootViewController as! FlutterViewController;
+    GeneratedPluginRegistrant.register(with: self);
+
+    let controller : FlutterViewController = window?.rootViewController as!FlutterViewController;
     let batteryChannel = FlutterMethodChannel.init(name: "samples.flutter.io/battery",
                                                    binaryMessenger: controller);
-  
     batteryChannel.setMethodCallHandler({
       (call: FlutterMethodCall, result: FlutterResult) -> Void in
       // Handle battery messages.
     });
-   
-    return true
+
+    return super.application(application, didFinishLaunchingWithOptions: launchOptions);
```

Comment:
We forgot to return `[super application:application didFinishLaunchingWithOptions:launchOptions]` in the Objective-C code.